lib/govuk_content_models/presentation_toggles.rb

```diff
@@ -28,35 +28,52 @@ def organ_donor_registration_key
     presentation_toggles['organ_donor_registration'] || self.class.default_presentation_toggles['organ_donor_registration']
   end
 
-  def promote_register_to_vote=(value)
-    value = value.is_a?(Boolean) ? value : value != '0' # if assigned using a checkbox
-    register_to_vote_key['promote_register_to_vote'] = value
-  end
-
-  def promote_register_to_vote
-    register_to_vote_key['promote_register_to_vote']
+  def promotion_choice=(value)
+    is_valid = ["none", "organ_donor", "register_to_vote"].include?(value)
+    if is_valid
+      promotion_choice_key["choice"] = value
+    end
+    # XXX: Should we throw if value is not valid?
```

Comment:
That's right. I generally prefer code that fails loudly when used incorrectly, but I don't know what the preferred pattern is.